### ISSUE TITLE
many to many, junction table.

### ISSUE THREAD

**Issue by cesarvargas00** [seegno/bookshelf-cascade-delete#13]
Hi, first of all, thank you for this plugin!! It's awesome! 🔥 

There's only one thing that I didn't find how to do and that is: when I destroy a model, if that model has `manyToMany` relationships, how do I proceed? for instance:

``` javascript
const User = bookshelf.Model.extend({
    tableName: 'users',
    hasTimestamps: true,

    addresses: function() {
        return this.hasMany('Location', 'user_id');
    },

    services: function() {
        return this.belongsToMany('Tag'); //n:m <--- this one right here
    },

    projects: function() {
        return this.hasMany('Project', 'user_id');
    },

    reviews: function() {
        return this.hasMany('Review', 'user_id');
    }
},{
    dependents:['addresses', 'services', 'projects', 'reviews']
});
```

there's a `tags_users` table in my database used as a _junction table_, but whenever I try to destroy this model, I get the following error:

`Unhandled rejection error: delete from "tags" where user_id IN ('5') - column "user_id" does not exist
    at Connection.parseE (/Users/cesar/dev/criarme/constroifacil/pinky/node_modules/pg/lib/connection.js:539:11)
    at Connection.parseMessage (/Users/cesar/dev/criarme/constroifacil/pinky/node_modules/pg/lib/connection.js:366:17)
    at Socket.<anonymous> (/Users/cesar/dev/criarme/constroifacil/pinky/node_modules/pg/lib/connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at TCP.onread (net.js:529:20)`

Any ideas? Thank you again! ( i'm not sure if this is the place to post such a doubt, but since I didn't find any better place, I just posted here :D )

**Comment by ricardogama** [comment]
Hey @cesarvargas00, I'll try to look into this issue in the next few days. Thanks for the report, cheers!

**Comment by ricardogama** [comment]
@cesarvargas00 Just released [1.0.3](https://github.com/seegno/bookshelf-cascade-delete/releases/tag/1.0.3) with a fix for this issue, hope it works!
